.github/workflows/ci.yml

```diff
@@ -30,6 +30,10 @@ jobs:
       - name: Install Docker buildx
         uses: docker/setup-buildx-action@v2
 
+      - name: Copy .env.example
+        run: cp .env.example .env
```

Comment:
Would need to update `.env.example` to `.env.local` if we want the tests to continue to work. Thanks for making these changes!